proto/cosmos/capability/types.proto

```diff
@@ -1,10 +1,9 @@
 syntax = "proto3";
-
-package cosmos_sdk.x.capability.v1;
+package cosmos.capability;
```

Comment:
Is `capability` just specific to IBC?

Reply:
Thoughts @cwgoes  ?

Reply:
It is not intended to be specific to IBC, this should be general, it can be used elsewhere in the SDK.

IBC also does not serialise capabilities in any protocol-standardised way.

Reply:
`cosmos.capability` is correct